app/models/video.py

```diff
@@ -0,0 +1,46 @@
+from flask import current_app
+from app.models.customer import Customer
+from app import db
+
+
+class Video(db.Model):
+    video_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    release_date = db.Column(db.DateTime)
+    total_inventory = db.Column(db.Integer)
+    available_inventory = db.Column(db.Integer)
+    current_renters = db.relationship('Customer', secondary='rental')                        
+
+    def convert_to_json(self):
+
+        response_body = {  
+            "id": self.video_id,
+            "title": self.title,
+            "release_date": self.release_date,
+            "total_inventory": self.total_inventory,
+            "available_inventory": self.available_inventory
+        }
+
+        return response_body
+
+
+# ❗️ Does it matter that I didn't give this its own file? 
```

Comment:
Python will accept it but people tend to pretty universally prefer each class to be in its own file in most programming languages for readability.